hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1068,10 +1072,11 @@ def try_shrinking_nodes(self, nodes, n):
             if node.was_forced:
                 return False  # pragma: no cover
 
-            if node.ir_type == "string":
+            if node.ir_type in {"string", "bytes"}:
+                size_kwarg = "min_size" if node.ir_type == "string" else "size"
                 # if the size *increased*, we would have to guess what to pad with
                 # in order to try fixing up this attempt. Just give up.
-                if node.kwargs["min_size"] <= attempt_kwargs["min_size"]:
+                if node.kwargs[size_kwarg] <= attempt_kwargs[size_kwarg]:
```

Comment:
reminds me that we wanted to switch the bytes implementation to be variable-size at some point, no rush though.

Reply:
yeah. I've got a personal todo for this somewhere, but that's not great practice so I added it to #3921.